src/core/tests/type_prop/is_nan.cpp

```diff
@@ -3,35 +3,32 @@
 //
 
 #include "gtest/gtest.h"
-#include "openvino/opsets/opset9.hpp"
 #include "openvino/opsets/opset10.hpp"
 #include "util/type_prop.hpp"
 
 using namespace std;
 using namespace ov;
```

Comment:
Consider add
```using namespace ov::opset10```

Then can create nodes without `opset` namespace and will be less effort to update e.g. `opset11`

Reply:
done